### ISSUE TITLE
[BUG] Death Knight - Runic Focus not providing 9% spell hit

### ISSUE THREAD

**Issue by Dassarian** [JamminL/cata-classic-bugs#921]
#### 1.1. Description
"Runic Focus" is a passive spell for Death Knights that is supposed to provide 9% spell hit in addition to double spell damage dealt. It does not seem to be granting the 9% spell hit.
  #### 1.2. How to Reproduce
           1. Need to be at the supposed Boss hit cap (17%).
           2. To get there we just need to gear for 8% hit. 
           3. After gearing for 8% hit the additional hit from "Runic Focus" should put me at / above the 17% spell hit cap. 
           4. Go to a boss target dummy and casts your DK spells at him, Icy Touch, Death Coil, etc.
  #### 1.3  Source Material
Just a link to wowhead to show the ability tooltip / text: (https://www.wowhead.com/cata/spell=61455/runic-focus)

**Comment by lordblackadder** [comment]
this one is either fixed, was never bugged, or is character specific issue. On the [following log](https://classic.warcraftlogs.com/reports/a:fqcN8Ava2nZHJ6y3) the first 2 fights are with 8% melee hit and had 0 misses in either IT, DC or Outbreak. The last fight was with 0% melee hit and it only took 1-2 minutes to start seeing misses

**Comment by Dassarian** [comment]
Your log shows some misses, if you had 17% shouldn't you have not missed at all? It is showing 11% miss chance - 3 misses of IT out of 25 casts. So it does seem off to me even looking at your own logs.

I am still experiencing high levels of misses. Actually very high, almost as if I have negative hit chance lol.

My log:
https://classic.warcraftlogs.com/reports/LzZH8hnPmKrfFvtM#fight=last&type=damage-done&source=4

Others in the DK discord have also said they are missing a lot. My character sheet shows 8.4% hit chance for melee (9.4% for spell). So I shouldn't be missing at all (oddly enough Heart Strike also missed at one point). 

I took off all my gear and put it back on too, just in case the hit wasn't being properly calculated from when I did the character copy, but that did not help.

**Comment by lordblackadder** [comment]
Yes, as I explained in my previous message the last fight was with 0% melee hit to force misses. The first 2 fights had 8% melee hit and have 0 misses. Is your character 85? 56% IT miss rate seems too high and it might indicate you are not max level and attacking the raid dummy which is lvl 88

**Comment by Dassarian** [comment]
Ah, I missed the other encounters, sorry about that. 

I did some research and found that the Master Target dummies do not follow the same rule as most other Skull level bosses (where they are calculated as your level +3). They are instead set at +3 levels above the current max level. I tested on a skull boss in Nax and MC, both resulted in no misses with the correct hit rating. Sorry for the false bug report!

So it does look to be working correctly. Thanks for checking into it!
